drivers/classic/src/mtclim_vic.c

```diff
@@ -38,16 +38,11 @@ int
 data_alloc(const control_struct *ctrl,
            data_struct          *data)
 {
-    int ok = 1;
-    int ndays;
+    int    ok = 1;
```

Comment:
I changed it.  I almost did a while back but decided not to get involved, it took 2 minutes and as you say, is closer to what the code means.